docs/attach-monitoring-tags.md

```diff
@@ -0,0 +1,52 @@
+# How to add 'AzureMonitor-Containers' Log Analytics workspace tags to Acs-engine Kubernetes cluster resources
+
+You can either use the Azure Powershell or Azure cli to attach the Log Analytics workspace tag to Acs-engine Kubernetes master nodes.
+
+These  tags required for the Azure Monitor for Containers Ux experience (https://docs.microsoft.com/en-us/azure/monitoring/monitoring-container-insights-overview )
+
+If you are not familiar with the concepts of azure resource tags (https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-using-tags)
+
+
+## Attach tags using Powershell
+
+Get the below powershell script files to your local computer.
+   - Powershell script file [AddMonitoringWorkspaceTags.ps1](https://github.com/Microsoft/OMS-docker/blob/ci_feature_prod/docs/acsengine/kubernetes/AddMonitoringWorkspaceTags.ps1)
+   - Refer for updating the Powershell execution policy (https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.security/set-executionpolicy?view=powershell-6)
+
+- Use the following PowerShell command from the folder containing the Powershell script file:
+
+``` sh 
+
+.\AddMonitoringWorkspaceTags.ps1 -SubscriptionId <Acs-Engine SubscriptionId> -ResourceGroupName <Acs-Engine ResourceGroup> -LogAnalyticsWorkspaceResourceId <WorkspaceResourceId>
+
+```
+
+The configuration change can take a few minutes to complete. When it finishes, you see a message something like this 'Successfully added logAnalyticsWorkspaceResourceId tag to K8s master VMs':
+
+## Attach tags using Azure CLI 
```

Comment:
Do you need to specify the same script again below ? Just point to the .sh may be ?

Reply:
Purged sh file and kept the az script in the doc.